src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -444,7 +444,6 @@ public Vector4 TextColor
                 {
                     SetInternalTextColor(value);
                 }
-                NotifyPropertyChanged();
```

Comment:
Do not remove it, because When NUIApplication.IsUsingXaml is true, NotifyPropertyChanged is not called